pkg/bff/members.go

```diff
@@ -78,16 +84,27 @@ func (s *Server) GetSummaries(ctx context.Context) (testnet *members.SummaryRepl
 
 // Overview endpoint is an authenticated endpoint that requires the read:vasp permission.
 func (s *Server) Overview(c *gin.Context) {
-	// TODO: Retrieve the user claims and retrieve the VASP details
+	var err error
+
+	// Get the bff claims from the context
+	var claims *auth.Claims
+	if claims, err = auth.GetClaims(c); err != nil {
+		log.Error().Err(err).Msg("unable to retrieve bff claims from context")
+		c.JSON(http.StatusInternalServerError, api.ErrorResponse(err))
+		return
+	}
+
+	// Extract the VASP IDs from the claims
+	testnetID := claims.VASP["testnet"]
+	mainnetID := claims.VASP["mainnet"]
```

Comment:
Is the expected behavior for the claims that this field will be empty if the registration hasn't been submitted for testnet or mainnet?

Reply:
Yes, that is the expected behavior - and there can be many reasons why the field would be empty; for example, the user may have to log out and log back in in order for the claims to be correctly populated.

Reply:
Any reason not to use the string constants for these?

Reply:
Good catch